css/card.css

```diff
@@ -59,6 +59,7 @@
   background-color: #fff;
   height: 90%;
   width: 90%;
+  z-index: 10;
```

Comment:
(nit) maybe comment explaining the z-index numbers, ie "above nav a.active" (or whatever would appear above it if this rule weren't specified)